app/questionnaire/path_finder.py

```diff
@@ -44,7 +44,21 @@ def routing_path(
         if section:
             for group in section["groups"]:
                 if "skip_conditions" in group:
-                    if evaluate_skip_conditions(
+                    if isinstance(group.get("skip_conditions"), dict):
+                        when_rule_evaluator = WhenRuleEvaluator(
+                            self.schema,
+                            self.answer_store,
+                            self.list_store,
+                            self.metadata,
+                            self.response_metadata,
+                            location=current_location,
+                            routing_path_block_ids=routing_path_block_ids,
+                        )
+                        when_rule = group.get("skip_conditions").get("when")
```

Comment:
Same as the comment for block, worth assigning skip conditions to a var.

Reply:
Assigned, thanks!